src/Core/Prefab/Globe/GlobeLayer.js

```diff
@@ -114,20 +114,16 @@ class GlobeLayer extends TiledGeometryLayer {
             return false;
         }
 
-        const points = node.obb.topPointsWorld;
+        // see https://cesiumjs.org/2013/04/25/Horizon-culling/
+        scaledHorizonCullingPoint.copy(node.horizonCullingPoint).applyMatrix4(worldToScaledEllipsoid);
+        scaledHorizonCullingPoint.sub(cameraPosition);
 
-        for (const point of points) {
-            // see https://cesiumjs.org/2013/04/25/Horizon-culling/
-            cullingVector.copy(point);
-            cullingVector.applyMatrix4(worldToScaledEllipsoid).sub(cameraPosition);
+        const vtMagnitudeSquared = scaledHorizonCullingPoint.lengthSq();
+        const dot = -scaledHorizonCullingPoint.dot(cameraPosition);
+        const isOccluded = magnitudeSquared < 0 ? dot > 0 : magnitudeSquared < dot && magnitudeSquared < ((dot * dot) / vtMagnitudeSquared);
 
-            const vtMagnitudeSquared = cullingVector.lengthSq();
-            const dot = -cullingVector.dot(cameraPosition);
-            const isOccluded = magnitudeSquared < dot && magnitudeSquared < ((dot * dot) / vtMagnitudeSquared);
-
-            if (!isOccluded) {
-                return false;
-            }
+        if (!isOccluded) {
```

Comment:
You can remove the `if` and write `return isOccluded`.

Reply:
changed